### DISCUSSION TITLE
dotnet build, restore, run FROZEN

### DISCUSSION THREAD

**Discussion by SoyDiego** [dotnet/aspnetcore#57306] [Q&A]
Hi I'm using Manjaro Linux and I was running my project using `dotnet run` and downloading the nugets from Azure doing `dotnet restore --interactive` but today I found some problems and I don't know what is happening...

![image](https://github.com/user-attachments/assets/b3049dc3-7331-4bee-99c2-81a9b8b383fc)

After that command nothing is happening. And If I cancel the command I'm receiving this warnings:

![image](https://github.com/user-attachments/assets/7673c400-d9d9-4537-b1d1-160bc7346aea)

## I have tried:
- Cloning my projects again
- Logout and login in Azure again
- Cleaning using `dotnet nuget locals all --clear` and `dotnet clear`

## `dotnet info`

```
.NET SDK:
 Version:           8.0.107
 Commit:            1bdaef7265
 Workload version:  8.0.100-manifests.43c23f91

Runtime Environment:
 OS Name:     manjaro
 OS Version:  
 OS Platform: Linux
 RID:         arch-x64
 Base Path:   /usr/share/dotnet/sdk/8.0.107/

.NET workloads installed:
 Workload version: 8.0.100-manifests.43c23f91
There are no installed workloads to display.

Host:
  Version:      8.0.7
  Architecture: x64
  Commit:       2aade6beb0

.NET SDKs installed:
  3.1.426 [/usr/share/dotnet/sdk]
  7.0.120 [/usr/share/dotnet/sdk]
  8.0.107 [/usr/share/dotnet/sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 3.1.32 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 7.0.20 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.AspNetCore.App 8.0.7 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.1.32 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 7.0.20 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  Microsoft.NETCore.App 8.0.7 [/usr/share/dotnet/shared/Microsoft.NETCore.App]
  ```
  
  Any help here or someone with the same problem?
  Or any idea what more can I try, please?
  
  Thanks!!

**Comment by martincostello** [comment]
Try running the command with `--verbosity detailed` to see where the time is being sent. A likely candidate is a slow NuGet package source.

**Reply by SoyDiego** [reply]
> --verbosity detailed

Hi, thanks for reply @martincostello. I have a lot of lines but I will share same of the last lines and after that it's frozen there:

![image](https://github.com/user-attachments/assets/510ae675-a290-4694-8917-d44a4b7af32b)

If you need something more, let me know and sorry If I cannot help so much, my knowledge in .NET is not so good

**Reply by SoyDiego** [reply]
> Try running the command with `--verbosity detailed` to see where the time is being sent. A likely candidate is a slow NuGet package source.

any idea @martincostello? Thanks in advance!
